internal/namespaces/account/v2alpha1/custom_test.go

```diff
@@ -1,72 +0,0 @@
-package account
-
-import (
-	"os"
-	"path"
-	"testing"
-
-	"github.com/scaleway/scaleway-cli/v2/internal/core"
-	account "github.com/scaleway/scaleway-sdk-go/api/account/v2alpha1"
-)
-
-func Test_initCommand(t *testing.T) {
```

Comment:
Could those tests be transfered in the iam package? I don't understand why they could not be adapted.